backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -100,6 +102,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.OfferAmount, src => src.OfferAmount)
                 .Map(dest => dest.Tasks, src => src.Tasks)
                 .Map(dest => dest.Responses, src => src.ProjectAgencyResponses)
+                .AfterMapping((src, dest) => dest.Metadata = JsonConvert.SerializeObject(dest)) //Map all non-ignored fields into the metadata.
```

Comment:
same idea, map the fields as per usual, allowing entity logic requiring fields that are stored in metadata to perform required checks. After mapping, serialize all [JsonProperty] fields to Metadata.